README.md

```diff
@@ -1,98 +1,79 @@
-* New deep learning framework
-* AutoML
-* Compiler technique for AI
-* High performance networking for AI
-* Profiling tool, including network, platform, and AI job profiling
-* AI Benchmark suite
-* New hardware for AI, including FPGA, ASIC, Neural Processor
-* AI Storage support
-* AI platform management
+Pre-built docker for popular AI frameworks. Easy to include heterogeneous hardware. Support Distributed training, such as distributed TensorFlow.
 
-PAI operates in an open model. It is initially designed and developed by [Microsoft Research (MSR)](https://www.microsoft.com/en-us/research/group/systems-research-group-asia/) and [Microsoft Search Technology Center (STC)](https://www.microsoft.com/en-us/ard/company/introduction.aspx) platform team.
-We are glad to have [Peking University](http://eecs.pku.edu.cn/EN/), [Xi'an Jiaotong University](http://www.aiar.xjtu.edu.cn/), [Zhejiang University](http://www.cesc.zju.edu.cn/index_e.htm), and [University of Science and Technology of China](http://eeis.ustc.edu.cn/) join us to develop the platform jointly.
-Contributions from academia and industry are all highly welcome.
-
-## System Deployment
-
-### Prerequisite
-
-The system runs in a cluster of machines each equipped with one or multiple GPUs.
-Each machine in the cluster should:
-1. Run Ubuntu 16.04 LTS.
-2. Assign a static IP address.
-3. Have no Docker installed or a Docker with api version >= 1.26.
-4. Have access to a Docker registry service (e.g., [Docker hub](https://docs.docker.com/docker-hub/))
-to store the Docker images for the services to be deployed.
-
-The system also requires a dev machine that runs in the same environment that has full access to the cluster.
-And the system need [NTP](http://www.ntp.org/) service for clock synchronization.
-
-### Deployment process
-To deploy and use the system, the process consists of the following steps.
-
-1. Deploy PAI following our [bootup process](./pai-management/doc/cluster-bootup.md)
-2. Access [web portal](./webportal/README.md) for job submission and cluster management
+### Most complete solution and easy to extend
 
+OpenPAI is a most complete solution for deep learning, support virtual cluster, compatible Hadoop / kubernetes eco-system, complete training pipeline at one cluster etc. OpenPAI is architected in a modular way: different module can be plugged in as appropriate. 
 
-#### Job management
+## How to deploy
+#### 1 Prerequisites
+Before start, you need to meet the following requirements:
 
-After system services have been deployed, user can access the web portal, a Web UI, for cluster management and job management.
-Please refer to this [tutorial](job-tutorial/README.md) for details about job submission.
+- Ubuntu 16.04
+- Assign each server a static IP address. Network is reachable between servers.
+- Server can access the external network. Have access to a Docker registry service (e.g., Docker hub) to store the Docker images for the services to be deployed.
```

Comment:
>Server can access the external network. Have access to a Docker registry service (e.g., Docker hub) to store the Docker images for the services to be deployed. [](start = 2, length = 159)

Server can access the external network, especially need to have access to a Docker registry service (e.g., Docker hub) to pull the Docker images for the services to be deployed.